arch/xtensa/Kconfig

```diff
@@ -103,6 +103,13 @@ config XTENSA_NUM_SPIN_RELAX_NOPS
 	  Specify the number of NOPs in Xtensa specific
 	  arch_spin_relax().
 
+config XTENSA_BREAK_ON_UNRECOVERABLE_EXCEPTIONS
+	bool "Use BREAK instruction on unrecoverable exceptions"
+	help
+	  Use BREAK instruction when unrecoverable exceptions are
+	  encountered. This requires a debugger attached to catch
+	  the BREAK.
```

Comment:
Do we really care to configure this?  All it costs is three bytes for the breakpoint instruction, and it's not like we would have done anything more useful anyway.

Reply:
There was a situation where this generated an interrupt storm when no debugger is attached. So might be a good idea to skip this for production firmware.